frontend/src/components/NotFound/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useCallback } from "react";
+import { useHistory } from "react-router-dom";
+
+import Button from "./styles/Button";
+import Text from "./styles/Text";
+import Container from "./styles/Container";
+import ButtonContainer from "./styles/ButtonContainer";
+import TextContainer from "./styles/TextContainer";
+
+import NotFoundImage from "Src/assets/NotFound";
+import ROUTES from "Src/constants/routes";
+
+const NotFound = (): JSX.Element => {
+	const { push } = useHistory();
+
+	const onClickHandler = useCallback(() => {
+		push(ROUTES.APPLICATION);
+	}, []);
+
+	return (
+		<Container>
+			<NotFoundImage />
+
+			<TextContainer>
+				<Text>Ah, seems like we reached a dead end!</Text>
+				<Text>Page Not Found</Text>
+			</TextContainer>
+
+			<ButtonContainer>
+				<Button tabIndex={0} onClick={onClickHandler}>
```

Comment:
We can use `Link` instead of updating route on click. Also can we return to the Homepage?

Reply:
Yes, that's another way of solving the problem.

Reply:
But right now refactored the code to make use of Link which re-route to the '/'

Reply:
> Yes, that's another way of solving the problem.

It will also make the link more accessible with screen reader